diesel/src/backend.rs

```diff
@@ -35,6 +37,8 @@ impl TypeMetadata for Debug {
     type TypeMetadata = ();
 }
 
+impl SupportsReturningClause for Debug {}
```

Comment:
Quite possibly nothing, as there's currently no case where we'll end up calling `as_query()` in the context of `backend::Debug`. However, I've been thinking about logging lately (see #150), and if we do end up using this for the eventual implementation, we'll want to generate the `RETURNING` clause there.